src/pycropml/transpiler/Parser.py

```diff
@@ -0,0 +1,50 @@
+import os
```

Comment:
Please, document the code. This is mandatory due to code complexity and maintenance.

What this module do?
What are the role of options (doc needed).
...